### PR TITLE
Add Mp3tag

### PR DESCRIPTION
**Pull request by chincheta0815**
Added cask for mp3tag.

### CODE REVIEW

File: Casks/mp3tag.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'mp3tag' do
+  version '2.79'
+  sha256 '15301427664658f3cd38f7156b1ff95891198d637a298baea68b236440581e80'
+
+  url "http://download.mp3tag.de/mp3tagv#{version.delete('.')}-MacOSX-Wine.zip"
+  name 'MP3TAG The universal tag editor.'
+  homepage 'http://www.mp3tag.de/en'
+  license :unknown # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
+
+  app "mp3tagv#{version.delete('.')}-MacOSX-Wine/Mp3tag.app", target: 'Mp3tag.app'
```
Comment (vitorgalvao) [inline Casks/mp3tag.rb]: `#{version.no_dots}` instead.

Also, remote the `target:`, it makes no sense in this case.

File: Casks/mp3tag.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'mp3tag' do
+  version '2.79'
+  sha256 '15301427664658f3cd38f7156b1ff95891198d637a298baea68b236440581e80'
+
+  url "http://download.mp3tag.de/mp3tagv#{version.delete('.')}-MacOSX-Wine.zip"
```
Comment (vitorgalvao) [inline Casks/mp3tag.rb]: `#{version.no_dots}` instead.

File: Casks/mp3tag.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name 'MP3TAG The universal tag editor.'
```
Comment (vitorgalvao) [inline Casks/mp3tag.rb]: This is not a place for descriptions. Make it just `MP3TAG`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by chincheta0815** [comment]
Requests merged.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**